fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/fluid/FluidVariant.java

```diff
@@ -34,8 +34,8 @@
  *
  * <p>{@link net.fabricmc.fabric.api.transfer.v1.client.fluid.FluidVariantRendering} can be used for client-side rendering of fluid variants.
  *
- * <p><b>Fluid variants must always be compared with {@link #equals}, never by reference!</b>
- * {@link #hashCode} is guaranteed to be correct and constant time independently of the size of the NBT.
+ * <p><b>Fluid variants must always be compared with {@code equals}, never by reference!</b>
+ * {@code hashCode} is guaranteed to be correct and constant time independently of the size of the NBT.
```

Comment:
Was that a javadoc error? The methods should exist in a superclass. I agree that linking is not very useful here.

Reply:
![image](https://user-images.githubusercontent.com/33279053/191036599-ce90e9e5-17fc-4ca3-a4e9-2407d5c65b9f.png)

Yes, Javadoc does not know that all implementers are `Object`.